src/Compilers/Core/Portable/Operations/OperationInterfaces.xml

```diff
@@ -2418,7 +2418,7 @@
       </Comments>
     </Property>
   </AbstractNode>
-  <Node Name="IMethodBodyOperation" Base="IMethodBodyBaseOperation" SkipInVisitor="true">
+  <Node Name="IMethodBodyOperation" Base="IMethodBodyBaseOperation" VisitorName="VisitMethodBodyOperation">
```

Comment:
why can't this visitor name be inferred from the name of the operation?

Reply:
Same as above.

Reply:
i don't understand this case though.  `IMethodBodyOperation <==> VisitMethodBodyOperation`.  i.e. the **MethodBodyOperation** case is the same for both, right?  So why can't this jsut be generated normally?

Reply:
Because the standard naming convention would be `VisitMethodBody` :(

Reply:
**click**.

thanks. :)